data/json/recipes/tools/tool.json

```diff
@@ -955,6 +975,20 @@
     "using": [ [ "plastic_molding", 1 ] ],
     "components": [ [ [ "plastic_chunk", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "gasket_rubber",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "20 m",
+    "proficiencies": [ { "proficiency": "prof_plasticworking" } ],
+    "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
```

Comment:
How does this have contain?   Does the other gasket have contain?  I would think contain is what things that use the gasket would have.